man/tpm2_changeauth.1.md

```diff
@@ -1,53 +1,68 @@
 % tpm2_changeauth(1) tpm2-tools | General Commands Manual
 %
-% SEPTEMBER 2017
+% OCTOBER 2018
 
 # NAME
 
-**tpm2_changeauth**(1) - Insert authorization values for the owner, endorsement
-and lockout authorizations.
+**tpm2_changeauth**(1) - Configuring authorization for TPM OWNER/ENDORSEEMNT/LOCKOUT hierarchies, NV Handles and Transient & Persistent object handles.
 
 # SYNOPSIS
 
 **tpm2_changeauth** [*OPTIONS*]
 
 # DESCRIPTION
 
-**tpm2_changeauth**(1) - set the various (owner, endorse, lockout)
-authorization values.
+**tpm2_changeauth**(1) - Managing authorization passwords for TPM objects/ handles.
+1. Permanent Handles (Owner, Endorsement, Lockout).
+2. Persistent Handles (0x81XX_XXXX).
+3. Transient Handles(0x80XX_XXXX).
+4. NV Handles(0x01XX_XXXX).
 
 # OPTIONS
 
-  * **-o**, **--owner-passwd**=_OWNER\_PASSWORD_:
-    
-    The new owner authorization value. Passwords should follow the "password 
-    authorization formatting standards", see section "Authorization Formatting".
+Passwords should follow the "password authorization formatting standards", see section "Authorization Formatting".
 
-  * **-e**, **--endorse-passwd**=_ENDORSE\_PASSWORD_:
+  * **-o**, **--new-owner-passwd**=_OWNER\_PASSWORD_:
+  * 
+    The new authorization value for the owner hierarchy . 
```

Comment:
The stray `*` on line 26 causes the description on line 27 to be an entry in the option list. Correct this.